setup.py

```diff
@@ -0,0 +1,1473 @@
+        'paddle.incubate.autograd',
+        'paddle.incubate.distributed',
+        'paddle.incubate.distributed.utils',
+        'paddle.incubate.distributed.utils.io',
+        'paddle.incubate.distributed.fleet',
+        'paddle.incubate.distributed.models',
+        'paddle.incubate.distributed.models.moe',
+        'paddle.incubate.distributed.models.moe.gate',
+        'paddle.sparse',
+        'paddle.sparse.nn',
+        'paddle.sparse.nn.layer',
+        'paddle.sparse.nn.functional',
+        'paddle.incubate.xpu',
+        'paddle.io',
+        'paddle.optimizer',
+        'paddle.nn',
+        'paddle.nn.functional',
+        'paddle.nn.layer',
+        'paddle.nn.quant',
+        'paddle.nn.initializer',
+        'paddle.nn.utils',
+        'paddle.metric',
+        'paddle.static',
+        'paddle.static.nn',
+        'paddle.static.amp',
+        'paddle.static.sparsity',
+        'paddle.tensor',
+        'paddle.onnx',
+        'paddle.autograd',
+        'paddle.device',
+        'paddle.device.cuda',
+        'paddle.device.xpu',
+        'paddle.version',
+        'paddle.profiler',
+        'paddle.geometric',
+        'paddle.geometric.message_passing',
+        'paddle.geometric.sampling',
+    ]
+
+    # packages=find_packages('python',exclude=['tools'])
+
+    # get scripts
+    paddle_bins = ''
+    if not envir_var.get("WIN32"):
+        paddle_bins = [
+            envir_var.get("PADDLE_BINARY_DIR") + '/paddle/scripts/paddle'
+        ]
+
+    # get package_data and package_dir
+    package_data, package_dir, ext_modules = get_package_data_and_package_dir()
+
+    # get headers
+    headers = get_headers()
+
+    return (
+        setup_requires,
+        packages,
+        paddle_bins,
+        package_data,
+        package_dir,
+        ext_modules,
+        headers,
+    )
+
+
+build_help_message = """
+    The following commands of setup.py may help you
+
+    if you want to install paddle and get the package that is stable and does not require you to edit, modify, or debug,plz run:
+        $ python setup.py install
+
+    if you want to install a package that requires you to modify it so that you have to reinstall it,plz run:
+        $ python setup.py develop
+
+    if you want to rerun cmake to re-generate native build files,plz run:
+        $ python setuo.py develop rerun-cmake
+
+    if you want to cmake only and then adjust build options by yourself, plz run:
+        $ python setuo.py develop only-cmake
+"""
+
+
+def print_info_of_reminding(help_messages):
+    print(help_messages)
+
+
+def main():
+
+    # Parse the command line and check arguments before we proceed with building steps and setup
+    parse_input_command(filter_args_list)
+
+    os.system('pip install ninja')
+    # Execute the build process,cmake and make
+    if cmake_and_make:
+        build_steps()
+
+    # envir_dict is environment variables after cmake
+    sys.path.append("/build/python/")
```

Comment:
This is an absolute path or relative path?

Reply:
both ok，We'd better use the absolute path，I wll fixt that problem